packages/roosterjs-content-model-plugins/lib/autoFormat/numbers/transformOrdinals.ts

```diff
@@ -41,3 +40,12 @@ export function transformOrdinals(
     }
     return false;
 }
+
+function getValue(text: string) {
+    const number = text.substring(0, text.length - 2);
```

Comment:
Suggest do not use hard coded number "2" here, but use a const and explain it with a comment.

Same for line 26